TidepoolOnboarding/Views/Sections/YourSettingsViews.swift

```diff
@@ -51,40 +59,59 @@ fileprivate struct YourSettingsTidepoolServiceOnboardingView: View {
     @State private var isSheetPresented = false
     @State private var onSheetDismiss: (() -> Void)?
 
+    @State private var deviceValid: Bool?
+    @State private var error: Error?
     @State private var skip = false
+    @State private var isNextButtonActing = false
```

Comment:
[nit] Just curious (because I had some confusion understanding the semantics of this): Is `isNextButtonActing` just `isNextButtonDisabled`, or is it more subtle than that?  If it isn't, then maybe consider `isNextButtonDisabled` for clarity?

Reply:
Effectively they are the same for this view, but other views that use a similar `isNextButtonActing` have other conditions that also set `nextButtonDisabled`.  The `isNextButtonActing` simply means that the next button is doing is its action which then has some impact on other controls (such as disabling the Next button, disabling the Birthday control, etc.)